VERSION.cmake

```diff
@@ -1,6 +1,6 @@
 set (DEFAULT_LIBDNF_MAJOR_VERSION 0)
 set (DEFAULT_LIBDNF_MINOR_VERSION 44)
-set (DEFAULT_LIBDNF_MICRO_VERSION 0)
+set (DEFAULT_LIBDNF_MICRO_VERSION 1)
```

Comment:
One more thing, though a formality mostly, I'd say we should increase the minor version when adding API, micro should be just for fixes?